students/ian_letourneau/mailroom/test_mailroomOO.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+# Ian Letourneau
+# 6/8/2018
+# Test script for mailroomOO.py
+
+import mailroomOO as m
+
+# Test Donor object creation
+LBdonations = [6578921.00, 3.50, 23400.00, 7234.00]
+BGdonations = [1235544.00, 456789.50, 2347899.75]
+JKdonations = [456.37, 25.67, 999876.00, 2134.78, 3.14]
+d1 = m.Donor("LeBron James", LBdonations)
+d2 = m.Donor("Bill Gates", BGdonations)
+d3 = m.Donor("Jimmy Kimmel", JKdonations)
+donors = m.DList(d1, d2, d3)
+
+
+def test_donor_creation():
+    assert d1.name == ("LeBron James")
+    assert d1.total == (6609558.5)
+
+# Test send_thank_you donor function
+
+
+def test_send_thank_you():
```

Comment:
your functions should have 2 lines space between them and if you want a comment for a function, it should go after the function definition:
```
def test_donor_creation():
    """Test send_thank_you donor function"""
    pass
```
but test function names are pretty self explanatory 🙂